IoT/Files/Parameters/IoTHubParameters.json

```diff
@@ -74,7 +74,8 @@
             ],
             "defaultValue": "S1",
             "properties": {
-                "attributes": "summarizable;"
+                "attributes": "summarizable;",
+                "summaryText": "{{{inputs.iotHubSku}}}"
```

Comment:
This change should not be required. If you do not specify any summaryText the input name is picked by default. SummaryText can be be used to specify some custom text e.g. dependent on some other input.

Reply:
Thanks for the feedback!

However, it looks like after removing the `summaryText` line, IoT Hub SKU no longer shows in the main screen:

![image](https://user-images.githubusercontent.com/716650/52936771-2f1d3680-3398-11e9-866c-1cf9e4dfa687.png)

Here is how it looks like before the change:

![image](https://user-images.githubusercontent.com/716650/52936489-40197800-3397-11e9-9569-ef00c8d4617b.png)

Reply:
yes, it seems I was wrong about that. You are good to go.